containers/ecr-viewer/src/app/layout.tsx

```diff
@@ -31,18 +31,20 @@ export default function RootLayout({
       <html lang="en">
         <head />
         <body className="overflow-x-auto">
-          <Header>
+        <AuthSessionProvider>
+        <div className="position-relative header-z-index isolate">
+        <Header>
```

Comment:
can these class names go on the header directly? doe we need more divs instead of having the line 40 div around everything in the body?